source/catalog.html

```diff
@@ -37,69 +43,80 @@ <h3>Анна</h3>
           <p>Здесь могла быть ваша цитата о высоком и вечном.</p>
           <div class="users-photo__like-zone">
             <button type="button" aria-label="Поставить лайк"></button>
             <span>Нравится: 215</span>
           </div>
+          </div>
         </div>
         <div class="users-photo__post">
-          <img src="" width="280" height="180" alt="Ну Краснодарский край как всегда">
-          <h3>Сергей</h3>
+          <a href="#"><img src="img/mobile/catalog/photo-4.png" width="280" height="180" alt="Ну Краснодарский край как всегда"></a>
+          <div class="users-photo__text-zone">
+          <h3>Сергей</h3><time datetime="1970-01-01 00:00">(25 часов назад)</time>
           <p>Самая кубанская в мире!</p>
           <div class="users-photo__like-zone">
             <button type="button" aria-label="Поставить лайк"></button>
             <span>Нравится: 4</span>
           </div>
+          </div>
         </div>
         <div class="users-photo__post">
-          <img src="" width="280" height="180" alt="Сколько тонн снега ради фотки не раскидаешь?">
-          <h3>Максим</h3>
+          <a href="#"><img src="img/mobile/catalog/photo-5.png" width="280" height="180" alt="Сколько тонн снега ради фотки не раскидаешь?"></a>
+          <div class="users-photo__text-zone">
+          <h3>Максим</h3><time datetime="1970-01-01 00:00">(40 часов назад)</time>
           <p>Где снег-то?</p>
           <div class="users-photo__like-zone">
             <button type="button" aria-label="Поставить лайк"></button>
             <span>Нравится: 150</span>
           </div>
+          </div>
         </div>
         <div class="users-photo__post">
-          <img src="" width="280" height="180" alt="Лечу снаружи самолёта и думаю о том, как же хорошо наверное лететь внутри">
-          <h3>Надежда</h3>
+          <a href="#"><img src="img/mobile/catalog/photo-6.png" width="280" height="180" alt="Лечу снаружи самолёта и думаю о том, как же хорошо наверное лететь внутри"></a>
+          <div class="users-photo__text-zone">
+          <h3>Надежда</h3><time datetime="1970-01-01 00:00">(45 часов назад)</time>
           <p>Всем GM и взаимные лайки!</p>
           <div class="users-photo__like-zone">
             <button type="button" aria-label="Поставить лайк"></button>
             <span>Нравится: 2560</span>
           </div>
+          </div>
         </div>
       </section>
       <section class="zone-download">
         <h2>Затмите их всех! Выкладывайте все, что накопилось в телефоне!</h2>
         <form class="zone-download__form" action="https://echo.htmlacademy.ru/" method="POST">
           <input class="visually-hidden" type="file" name="user-photo" id="user-photo-zone">
-          <label for="user-photo-zone">
-          <div class="zone-download__example-photo"></div>
-          </label>
+          <label class="zone-download__example-photo for="user-photo-zone"></label>
```

Comment:
```suggestion
          <label class="zone-download__example-photo" for="user-photo-zone"></label>
```
![image](https://user-images.githubusercontent.com/17790771/128057791-68a57c32-6687-4e12-bad2-e24c4178b05f.png)